### DISCUSSION TITLE
Change Cart Image

### DISCUSSION THREAD

**Discussion by rodigu** [nesbox/TIC-80#1835] [Q&A] [answered]
How do I change the cart image?
When I export it just has a blank cover.

**Comment by nesbox** [comment, marked as answer]
Run the game press `F7` and save then, or use the `import screen %file.png%` command.

**Reply by LEMONTECHyt** [reply]
how do i do it on android

**Reply by Skeptim** [reply]
I think there is the F7 key on the TIC80 android keyboard, but if not you can find other solutions there: https://github.com/nesbox/TIC-80/wiki/Cover-image
